models/patchmatch.py

```diff
@@ -183,9 +181,8 @@ def __init__(self, G: int = 8, stage: int = 3, evaluate_neighbors: int = 9, iter
 
         self.G = G
         self.stage = stage
-        if self.stage == 3:
-            self.pixel_wise_net = PixelwiseNet(self.G)
-
+        self.pixel_wise_net = PixelwiseNet(self.G)

```

Comment:
Yes, TorchScript cannot handle conditional members. It can be different between the branches, but it must exist for all branches. This effectively leads to a few unused members, but the functionality is not otherwise impacted.